src/SpeciesElements.f90

```diff
@@ -101,67 +90,57 @@ FUNCTION getElementArray(this, formula) RESULT(eleArray)
    ENDFUNCTION  getElementArray
 !
 !-------------------------------------------------------------------------------
-!> @brief Determins whether a character is in the alphabet using ASCII format
+!> @brief Determines whether a character is in the alphabet using ASCII format
 !> @param letter character to check
```

Comment:
I missed this the first time, but these headers are incomplete.  Need to add a line that has something `!> @returns isValid <description>` where you fill in the description.  You also would normally need a `!> @param this <description>` line as well, but since you're making these standalone functions instead of typebound, that will no longer apply here.

Need to make this change to all 3 of `isChar`, `isCharCap`, and `isCharLower`

Reply:
I added more to the headers